docs/DeveloperGuide.md

```diff
@@ -86,15 +92,15 @@ Guarantees: Editing of internship application is successful
   1b1. InternHunter displays an error message and informs the user of the valid input format. <br />
   Use case resumes from step 1.
 
-### Use case: UC10 - Delete a skill or achievement
+### Use case: UC10 - Delete a user profile item 
 
 #### MSS
 
-Precondition: User already has an existing list of skills and achievements. </br>
-Guarantees: Deletion of skill or achievement is successful
+Precondition: User already has an existing user profile item. </br>
```

Comment:
just a nitpick but I noticed that other use cases use list user profile items instead user profile item?

Reply:
Okay, will update!